lib/origen/commands.rb

```diff
@@ -62,24 +62,31 @@
   Origen.log.info 'Started code coverage'
   SimpleCov.configure do
     filters.clear # This will remove the :root_filter that comes via simplecov's defaults
+
     add_filter do |src|
-      !(src.filename =~ /^#{Origen.root}\/lib/)
+      if File.directory?("#{Origen.root}/app/lib")
+        !(src.filename =~ /^#{Origen.root}\/app\/lib/)
+      else
+        !(src.filename =~ /^#{Origen.root}\/lib/)
+      end
     end
 
     # Results from commands run in succession will be merged by default
     use_merging(!ARGV.delete('--no_merge'))
-
     # Try and make a guess about which directory contains the bulk of the application's code
     # and create groups to match the main folders
     d1 = "#{Origen.root}/lib/#{Origen.app.name.to_s.underscore}"
     d2 = "#{Origen.root}/lib/#{Origen.app.namespace.to_s.underscore}"
     d3 = "#{Origen.root}/lib"
+    d4 = "#{Origen.root}/app/"
```

Comment:
I think we should make this highest prio, for an app under the new structure the app dir should contain the majority of their code even if they happen to have an old style lib/ dir for some reason

Reply:
sounds good!

Reply:
@ginty pushing updates now